metadata.json

```diff
@@ -55,15 +54,15 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "6",
-        "7"
+        "7",
+        "8"
```

Comment:
Not a problem, but we shouldn't forget to make issues for this so it's clear in the changelog.

Reply:
ok will do